sdk/resourcemanager/azure-resourcemanager-eventhubs/src/main/java/com/azure/resourcemanager/eventhubs/implementation/EventHubNamespaceImpl.java

```diff
@@ -252,6 +254,16 @@ public PagedIterable<EventHubNamespaceAuthorizationRule> listAuthorizationRules(
                 .listByNamespace(this.resourceGroupName(), this.name());
     }
 
+    @Override
+    public TlsVersion minimumTlsVersion() {
+        return this.innerModel().minimumTlsVersion();
+    }
+
+    @Override
+    public Boolean zoneRedundant() {
+        return Objects.isNull(this.innerModel().zoneRedundant()) ? false : this.innerModel().zoneRedundant();
```

Comment:
ResourceManagerUtils.toPrimitiveBoolean

Reply:
return `boolean`, **lowercase b**, Please just read the comment.

Reply:
Fixed in the new version.